src/main/resources/init-script-gradle.ftl

```diff
@@ -101,6 +102,11 @@ gradle.allprojects {
 // ## START methods invoked by tasks above
 <#-- Do not parse with Freemarker because Groovy variable replacement in template strings is the same as Freemarker template syntax. -->
 <#noparse>
+def isRoot(Project project) {
```

Comment:
Could root project and a sub-project have the same name?

Reply:
If such a case did come up with a user that has enabled this property, suggesting they do not name their root project the same as a subproject would be good advice and I would strongy argue that is not a use case we should tip-toe around.